test/dotcom/alerts/disruptions/subway_test.exs

```diff
@@ -119,6 +270,7 @@ defmodule Dotcom.Alerts.Disruptions.SubwayTest do
       assert %{today: [^alert_today]} = todays_disruptions()
     end
 
+    @tag :skip
```

Comment:
Does this need to be skipped?

Reply:
Ohhh woops! No, this needs to be repaired, I think

Reply:
[PTALA](https://github.com/mbta/dotcom/pull/2402/commits/a9352bfdad7ddb19104e2ab988ed873597436dee)